src/controls/__snapshots__/Controls.spec.js.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
good job with the snapshot. I know the instructions on this were not the best but we covered it in lecture so I'm glad you made this.